summertime/model/single_doc/multilingual/mbart_model.py

```diff
@@ -93,7 +93,7 @@ def summarize(self, corpus, queries=None):
 
         encoded_summaries = self.model.generate(
             **batch,
-            decoder_start_token_id=self.tokenizer.lang_code_to_id[lang_code],
+            forced_bos_token_id=self.tokenizer.lang_code_to_id[lang_code],
```

Comment:
Hmm, what's the difference?

Reply:
don't think there is a difference, will switch back to decoder_start_token_id because its name is more self explanatory imo